src/test/java/io/confluent/connect/elasticsearch/integration/ElasticsearchConnectorBaseIT.java

```diff
@@ -15,6 +15,23 @@
 
 package io.confluent.connect.elasticsearch.integration;
 
+import com.google.gson.JsonArray;
```

Comment:
change in the codestyle?

Reply:
hey @pbadani i didn't get you. I did not change the codestyle, any specific line you think is ill-formatted?

Reply:
the imports were reorganized, nothing else.

Reply:
oh, gotcha. I'll revert those back 👍